.gitignore

```diff
@@ -131,3 +132,11 @@ venv.bak/
 dmypy.json
 
 *.cproject
+
+# generated parser
+src/scenic/syntax/parser.py
+
+# example output files
+examples/webots/city_intersection/*
```

Comment:
There doesn't seem to be any such folder.

Reply:
Ah good catch, I renamed it in the meantime. I'll ad a local gitignore to avoid this in the future.